curated_transformers/tests/tokenization/test_wordpiece_encoder.py

```diff
@@ -71,3 +81,27 @@ def test_serialize_hf_model():
         encoder.attrs["wordpiece_processor"].to_list()
         == encoder2.attrs["wordpiece_processor"].to_list()
     )
+
+
+def _test_toy_encoder(encoder):
+    nlp = spacy.blank("en")
+    doc1 = nlp.make_doc("I saw a girl with a telescope.")
+    doc2 = nlp.make_doc("Today we will eat poké bowl.")
```

Comment:
I feel like this could be made a fixture (since we use this in multiple tests across different packages)?

Reply:
Done. I didn't use the fixture in `test_pipe`, because we need the `nlp` object in those tests.